doc/content/getting-started/packet-broker/api.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Mapper API
+description: ""
+---
+
+
+The Packet Broker Mapper API can help you fetch gateway locations, online status and other gateway information for {{% tts %}} Community Edition, Cloud, Enterprise and Open Source deployments. This section contains the reference for the Packet Broker Mapper API and examples of forming an API call.
+
+<!--more-->
+
+The [OpenAPI](https://swagger.io/specification/) definition of the Packet Broker Mapper API is available [here](https://mapper.packetbroker.net/api/v2/openapi.json). You can use online Swagger editors like [Petstore](https://petstore.swagger.io/) to explore the Mapper API.
```

Comment:
Maybe good to make it explicit what URL they have to paste and then click **Explore**

```suggestion
The [OpenAPI](https://swagger.io/specification/) definition of the Packet Broker Mapper API is available [here](https://mapper.packetbroker.net/api/v2/openapi.json). You can also use [Swagger UI](https://petstore.swagger.io/) to explore the Mapper API by entering `https://mapper.packetbroker.net/api/v2/openapi.json` in the URL and clicking **Explore**.
```